File: api/src/main/java/io/github/wulkanowy/api/notes/Note.java
package io.github.wulkanowy.api.notes;

public class Note {

    private String date;

    private String teacher;

    private String category;

    private String content;

    public String getDate() {
        return date;
    }

    public Note setDate(String date) {
        this.date = date;
        return this;
    }

    public String getTeacher() {
        return teacher;
    }

    public Note setTeacher(String teacher) {
        this.teacher = teacher;
        return this;
    }

    public String getCategory() {
        return category;
    }

    public Note setCategory(String category) {
        this.category = category;
        return this;
    }

    public String getContent() {
        return content;
    }

    public Note setContent(String content) {
        this.content = content;
        return this;
    }
}

File: app/src/main/java/io/github/wulkanowy/ui/modules/debug/notification/mock/note.kt
package io.github.wulkanowy.ui.modules.debug.notification.mock

import io.github.wulkanowy.data.db.entities.Note
import io.github.wulkanowy.sdk.scrapper.notes.NoteCategory
import java.time.LocalDate

val debugNoteItems = listOf(
    generateNote("Aleksadra Krajewska", "Przeszkadzanie na lekcjach", NoteCategory.NEGATIVE),
    generateNote("Zofia Czerwińska", "Udział w konkursie szkolnym", NoteCategory.POSITIVE),
    generateNote("Stanisław Krupa", "Kultura języka", NoteCategory.NEUTRAL),
    generateNote("Karolina Kowalska", "Wypełnianie obowiązków ucznia", NoteCategory.NEUTRAL),
    generateNote("Joanna Krupa", "Umycie tablicy cifem", NoteCategory.POSITIVE),
    generateNote("Duchowicz Maksymilian", "Reprezentowanie szkoły", NoteCategory.POSITIVE),
    generateNote("Michał Mazur", "Przeszkadzanie na lekcji", NoteCategory.NEGATIVE),
    generateNote("Karolina Kowalska", "Wypełnianie obowiązków ucznia", NoteCategory.NEGATIVE),
    generateNote("Aleksandra Krajewska", "Wysadzenie klasy w powietrze", NoteCategory.NEGATIVE),
)

private fun generateNote(teacher: String, category: String, type: NoteCategory) = Note(
    teacher = teacher,
    category = category,
    categoryType = type.id,
    studentId = 0,
    date = LocalDate.now(),
    teacherSymbol = "",
    isPointsShow = false,
    points = 0,
    content = ""
)
